Install ESAPI extension for testbox run

michaelborn opened the pull request:
Running `testbox run` on this minibox-ci image requires the ESAPI extension to be installed.

michaelborn committed:
Install ESAPI extension for testbox run

Running `testbox run` on this minibox-ci image requires the ESAPI extension to be installed.

bdw429s commented:
Using the env var will be more future proof since this locks in a version of the extension.

michaelborn committed:
improve: Use env var to install latest ESAPI extension

This improves my PR for ESAPI support in cfml-ci-tools.

Using the `LUCEE_EXTENSIONS` environment variable means that as soon as any `box` command is run, the latest version of that extension is installed into the on-container CF engine.

michaelborn commented:
Closing this PR... don't know why I created a PR for my own fork :joy:
